### PR TITLE
Fix collect tests and content packs 

### PR DESCRIPTION
**Pull request by dantavori**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Replaced DocumentationTest with HelloWorld-Test in case no tests were collected.
- Added DeveloperTools to packs_to_install and moved TestCommonPython to DeveloperTools pack.

### CODE REVIEW

File: Tests/scripts/collect_tests_and_content_packs.py [hunk 1]
```diff
@@ -1116,11 +1116,16 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, two_be
             print_warning('Collecting sample tests due to: {}'.format(','.join(sample_tests)))
         else:
             print_warning("Running Sanity check only")
-            tests.add('DocumentationTest')  # test with integration configured
+
             tests.add('TestCommonPython')  # test with no integration configured
+            packs_to_install.add("DeveloperTools")
```
Comment (yaakovi) [inline Tests/scripts/collect_tests_and_content_packs.py:1121]: why do you need this? aren't we always installing that pack?

File: Tests/scripts/collect_tests_and_content_packs.py [hunk 2]
```diff
@@ -1116,11 +1116,16 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, two_be
...
+
+            tests.add('HelloWorld-Test')  # test with integration configured
+            packs_to_install.add("HelloWorld")
 
     if changed_common:
         tests.add('TestCommonPython')
+        packs_to_install.add("DeveloperTools")
```
Comment (yaakovi) [inline Tests/scripts/collect_tests_and_content_packs.py:1128]: same

File: Tests/scripts/collect_tests_and_content_packs.py (hunk 1 above)
Comment (IkaDemisto) [reply Tests/scripts/collect_tests_and_content_packs.py:1121]: Installing but not re-uploading. @yaakovi see my comment on second pr

**Comment by dantavori** [comment]
@yuvalbenshalom the build passed except for the validate errors which I opened an [issue](https://github.com/demisto/etc/issues/25480) about.

**Pull request merged**